src/actions/recomputeReduxState.js

```diff
@@ -92,13 +92,29 @@ const modifyStateViaURLQuery = (state, query) => {
   if (query.tl) {
     state["tipLabelKey"] = query.tl===strainSymbolUrlString ? strainSymbol : query.tl;
   }
+  
   if (query.dmin) {
-    state["dateMin"] = query.dmin;
-    state["dateMinNumeric"] = calendarToNumeric(query.dmin);
+    const dateNum = calendarToNumeric(query.dmin);
+    if (dateNum===undefined || dateNum < state.absoluteDateMinNumeric || dateNum > state.absoluteDateMaxNumeric) {
+      console.error(`Requested dmin (min date) "${query.dmin}" is invalid`);
```

Comment:
_non-blocking_

Align this error message with the max date one. 

```suggestion
      console.error(`Requested min date "${query.dmin}" is invalid`);
```

Reply:
I've made the error messages more consistent + expanded them as per Jover's comment